src/Article.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { getArticleById } from "./utils";
+import CommentSection from "./CommentsSection";
+import { Box } from "@mui/material";
+import FullArticleCard from "./FullArticleCard";
+
+export default function Article() {
+  const [article, setArticle] = useState([]);
```

Comment:
We should try to make sure the initial state should reflect the type that we expect this to be used as - e.g. once the request fulfils we expect article to be an object (I suspect as you are accessing the properties as article.body etc)